src/SonarQube.Client/ChecksumCalculator.cs

```diff
@@ -36,14 +36,18 @@ public interface IChecksumCalculator
     /// </summary>
     public class ChecksumCalculator : IChecksumCalculator
     {
+        private static readonly Regex StripWhitespaceRegEx = new Regex("\\s",
+            RegexOptions.Compiled, 
+            TimeSpan.FromMilliseconds(300));
```

Comment:
Another class in `SonarQube.Client`, so again I used a hard-coded timeout.
I thought about adding a separate timeout constant for `SonarQube.Client` but decided it wasn't worth it with only two instances.

The correct fix would be to refactor the code to reverse the dependency between `SonarQube.Client` and `Core`, but that's a much bigger job.